en/12_Project_Governance/05_Release_policy.md

```diff
@@ -0,0 +1,191 @@
+- **Silverstripe CMS major version series**: All Silverstripe CMS versions that start with the same whole number, e.g. "6.0", "6.1", and "6.2" are all in the same "CMS 6" major release series
+
+### Pre-release definitions
+
+- **Alpha**: A pre-release that may be performed prior to a "beta pre-release". Its purpose is to allow for early exploration and feedback on foundational changes, new architectural approaches, or significant new features
+- **Beta**: A pre-release is performed prior to a "RC pre-release". Its purpose is to facilitate testing and feedback, allow early adopters to plan their upgrade, to identify bugs, and to help community modules dependent on Silverstripe CMS to upgrade ahead of the stable release
+- **Release candidate (aka RC)**: A pre-release that is performed prior to a stable release. A copy of the release candidate (which also includes any undisclosed security patches) will be sent to an external auditor for a security review
+
+## Types of Silverstripe CMS releases
+
+Regular Silverstripe CMS releases are performed at predefined intervals with a clear support timeline. This allows Silverstripe CMS project owners to plan upgrades ahead of time.
+
+> [!NOTE]
+> The cadence and pre-release timeframes are indicative only. We may alter those in response to external constraints.
+
+### Silverstripe CMS major release
+
+A new Silverstripe CMS major version series (e.g. "CMS 6") is introduced by its first Silverstripe CMS release (e.g. "CMS 6.0").
+
+The main purpose of a major release is to deliver significant architectural updates and changes that are not backward compatible. This ensures the Silverstripe CMS major version series can be supported for its full lifecycle, including updates to third-party dependencies.
+
+At launch, new Silverstripe CMS major versions may contain new features not present in the previous major version series. However, shipping new features is a secondary concern, as new features will usually ship in Silverstripe CMS minor releases.
+
+- **Backward compatibility:** No - projects will require some development work to upgrade to a new major release
+- **Cadence:** Every two years, between April and June
+- **Pre-release schedule:**
+  - **Alpha**: At the discretion of the development team
+  - **Beta**: At least 3 months prior to the anticipated Silverstripe CMS major release. At this point, dependencies should not change
+  - **RC**: Approximately 4 weeks prior to the anticipated Silverstripe CMS major release. At this point, no more breaking changes can be made unless they are to fix a regression
+- **Documentation:** A changelog is published listing significant new features and breaking changes. A [clear upgrade path](/upgrading) is available
+
+### Silverstripe CMS minor release
+
+A Silverstripe CMS minor release is the standard recurring release of Silverstripe CMS (e.g. "CMS 6.1", "CMS 6.2"). Minor release upgrades are generally more straightforward to manage than major release upgrades.
```

Comment:
```suggestion
A Silverstripe CMS minor release is the standard recurring release of Silverstripe CMS (e.g. "CMS 6.1.0", "CMS 6.2.0"). Minor release upgrades are generally more straightforward to manage than major release upgrades.
```
I think technically since we defined the "version" as x.y, the release would be identified by its full tag name.